docs/deploy/rock-on-rhel.md

```diff
@@ -0,0 +1,185 @@
+# Rock on RHEL
+We are ofthen asked how deploy rock on RHEL. This should work for both a single node deployment and a multi-node deployment.

```

Comment:
Link to https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux and call it by its formal name before using the acronym.